src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/Channel.orm.xml

```diff
@@ -17,6 +17,12 @@
                                       http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
 
     <mapped-superclass name="Sylius\Component\Core\Model\Channel" table="sylius_channel">
+        <many-to-one field="defaultLocale" target-entity="Sylius\Component\Locale\Model\LocaleInterface">
+            <join-column name="default_locale_id" referenced-column-name="id" nullable="true" on-delete="SET NULL"/>
+        </many-to-one>
+        <many-to-one field="defaultCurrency" target-entity="Sylius\Component\Currency\Model\CurrencyInterface">
+            <join-column name="default_currency_id" referenced-column-name="id" nullable="true" on-delete="SET NULL"/>
+        </many-to-one>
```

Comment:
For now let's keep them nullable, but in the end, they should not be nullable. But that requires a bit more work, we will get there. :)